tests/bootstrap.php

```diff
@@ -1,4 +1,9 @@
 <?php
+
+namespace {
+    setlocale(LC_ALL, 'C');
```

Comment:
Do we want all tests running in this locale?

Reply:
Currently Guzzle fails on other locales. See guzzle/psr7#269 for details, it also solves this.